src/Twig/HtmlUtilitiesExtension.php

```
@@ -45,4 +53,26 @@ public function buildCssClasses(array $cssClassTests = []): string
         }
         return trim(implode(' ', $cssClasses));
     }
+
+    public function addScriptSnippet(string $snippet)
+    {
+        if ($snippet) {
```

Hmmm, not sure we should allow empty strings, maybe throw an exception. @BjornTwachtmann any thoughts?